eng/testing/tests.targets

```diff
@@ -43,7 +43,7 @@
     <_withoutCategories Condition="'$(WithoutCategories)' != ''">;$(WithoutCategories.Trim(';'))</_withoutCategories>
 
     <TestScope Condition="'$(TestScope)' == '' and '$(Outerloop)' == 'true'">all</TestScope>
-    <_withCategories Condition="'$(TestScope)' == 'outerloop'">$(_withCategories);OuterLoop</_withCategories>
+    <_withCategories Condition="'$(TestScope)' == 'all'">$(_withCategories);OuterLoop</_withCategories>
```

Comment:
Could you please explain how this is will fix the issue ?

Reply:
When we run test suite passing `/p:TestScope=outerloop` it ignores all attributes (`SkipOnPlatform`, `ActiveIssue`). But I noticed that works fine when we pass `/p:OuterLoop=true`. 
As `Outerloop=true` and `TestScope=outerloop` should behave same way and in line 45 we set `TestScope=all` I changed it to check for `all` not `outerloop`.
After this change it takes all categories and tests are skipped properly in the pipeline.

Reply:
> Maybe it would be better to check for both `all` and `outerloop`?

Sounds better to me, because I don't know what other places are setting `all` or `outerloop`

Reply:
Updated.